### DISCUSSION TITLE
Alphabet Does Not Go A to Z

### DISCUSSION THREAD

**Discussion by V0ldek** [V0ldek/Sorcery#8] [Sourcery]
We're going to examine a seemingly trivial task – look at some text and tell if it contains only *alphabetic characters*, or, in other words, *letters*. Simlarly, we might want to check if the text contains any letters, begins with a letter, etc. In any case, the root issue is telling whether a given character is a letter. **And you're probably doing it wrong**.

New Sourcery shenanigans available at:
https://v0ldek.com/sourcery/alphabet-does-not-go-a-to-z

**Comment by piotr-lewandowski** [comment]
Regexes were a mistake
